src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -196,25 +225,8 @@ private void waitFor(final Predicate<AutoplayVuap> predicate, final long timeout
         .until(predicate);
   }
 
-  private void clickElement(final String selector) {
-    WebElement element = wait.forElementClickable(By.cssSelector(selector));
-    Actions builder = new Actions(driver);
-    builder.contextClick(element)
-        .moveToElement(element)
-        .click(element)
-        .perform();
-  }
-
-  private WebElement findSpeakerIcon() {
-    return wait.forElementClickable(By.cssSelector(String.format(SPEAKER_SELECTOR_FORMAT, slot)));
-  }
-
-  private WebElement findCloseButton() {
-    return wait.forElementClickable(By.cssSelector(String.format(CLSE_BUTTON_SELECTOR_FORMAT, slot)));
-  }
-
   private boolean isOverlayNoVisible() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "isOverlayNoVisible" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)